test/csuite/timestamp_abort/main.c

```diff
@@ -81,9 +81,11 @@ static const char *const ckpt_file = "checkpoint_done";
 static bool compat, inmem, stress, use_ts;
 static volatile uint64_t global_ts = 1;
 
-#define ENV_CONFIG_COMPAT ",compatibility=(release=\"2.9\")"
+#define ENV_CONFIG_ADD_COMPAT ",compatibility=(release=\"2.9\")"
+#define ENV_CONFIG_ADD_STRESS ",timing_stress_for_test=[prepare_checkpoint_delay]"
 #define ENV_CONFIG_DEF                                        \
-    "cache_size=20M,create,"                                  \
+    "cache_size=%" PRIu32                                     \
+    "M,create,"                                               \
     "debug_mode=(table_logging=true,checkpoint_retention=5)," \
     "eviction_updates_trigger=95,eviction_updates_target=80," \
```

Comment:
One of the things Alex talked about in the ticket is adjusting the `eviction_dirty` settings too. Admittedly, the distinction between `eviction_dirty_*` and `eviction_updates_*` is subtle and kind of lost on me. So I'm not sure if we want one, the other or both.

Reply:
@sueloverso Thanks!  I've reviewed and updated the targets/triggers, and adjusted the comments and the calculation.  Turned out I didn't really understand them either!  Now I think I get it.  Can you take another look, please?  I'll have Alex take a look also.

Reply:
@sulabhM, could you please take a look at this change and see if the calculation and comment make sense? Want to make sure this makes sense in the world of eviction targets and triggers.  Thanks!

Reply:
Thank you! The new comment is very helpful.